### PR TITLE
CLIMATE-632 Merra loader

### PR DESCRIPTION
**Pull request by huikyole**
https://issues.apache.org/jira/browse/CLIMATE-632

The loader has been tested with 3-hourly MERRA reanalysis data.

### CODE REVIEW

File: ocw/data_source/local.py [hunk 1]
```diff
@@ -109,6 +111,61 @@ def _get_netcdf_variable_name(valid_var_names, netcdf, netcdf_var):
...
+    :returns: An OCW Dataset object with the requested variable's data from
+        the NetCDF file.
+    :rtype: :class:`dataset.Dataset`
+    :raises ValueError: 
+    '''                  
+    
+    merra_files = []
+    for pattern in filename_pattern:
+        merra_files.extend(glob(file_path + pattern))
+    merra_files.sort()
```
Comment (kwhitehall) [inline ocw/data_source/local.py]: @huikyole  you can save on this for loop with the following one liner
merra_files=glob.glob(file_path+'/'+filename_pattern+'*')

**Comment by kwhitehall** [comment]
This merge failed because of the following import
from pyhdf.SD import SD
@MJJoyce  can you please advice here? I

**Comment by MJJoyce** [comment]
Hey @kwhitehall, that library isn't a dependency that we have listed for the codebase. So that library will need to be added as a dependency for easy-ocw in a separate pull request and any relevant documentation will need to be updated as well. Then we can update the jenkins build to include this new dependency and the build will pass here. That would be my suggestion.

**Comment by MJJoyce** [comment]
@huikyole please update the title of this pull request so it mirrors properly to JIRA per the [developer guide on the wiki](https://cwiki.apache.org/confluence/display/CLIMATE/Developer+Getting+Started+Guide#DeveloperGettingStartedGuide-Step-By-StepDirections).

Comments and such don't mirror properly to JIRA if the title isn't just right. The project standard is:

CLIMATE-TicketNumber - Title of ticket

Thanks much

**Pull request closed without merging**